index.js

```diff
@@ -380,28 +331,11 @@ if (isJson) {
     })
   }
 
-  Log.prototype.silly = function silly() {
+  Log.prototype.silly = function silly(msg, meta) {
     if (!this._shouldLog('silly')) return
-
-    let str
-    switch (arguments.length) {
-      case 1:
-        str = format(arguments[0])
-        break
-      case 2:
-        str = format(arguments[0], arguments[1])
-        break
-      case 3:
-        str = format(arguments[0], arguments[1], arguments[2])
-        break
-      default:
-        const args = new Array(arguments.length)
-        for (var i = 0; i < args.length; i++) {
-          args[i] = arguments[i]
-        }
-        str = format.apply(util, args)
-        break
-    }
+    const str = arguments.length === 1
```

Comment:
instead of checking arguments length here. maybe use rest in the method signature? `Log.prototype.silly = function silly(...data)`, then check the length of data?